src/test/java/de/adesso/kicker/user/UserControllerTest.java

```diff
@@ -0,0 +1,48 @@
+package de.adesso.kicker.user;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+public class UserControllerTest {
+
+    private MockMvc mockMvc;
+
+    private UserDummy userdummy = new UserDummy();
+
+    @Mock
+    private UserService userService;
+
+    @InjectMocks
+    private UserController userController;
+
+    @Before
+    public void setUp() throws Exception {
+        mockMvc = MockMvcBuilders.standaloneSetup(userController).build();
```

Comment:
If we autowire `MockMvc`, and annotate the class with `@WebMvc(UserController.class)` and use the `@MockBean` the templates will be rendered, which gives us feedback on whether all templates are functional. Alternatively you can try properly configure it with the `MockMvcBuilders`.